### DISCUSSION TITLE
Cannot move beyond Step 1

### DISCUSSION THREAD

**Discussion by blindaks** [skills/.github#374] [Introduction to GitHub] [answered]
I have done the task for Step 1. But I cannot move to step 2. Please help me out.

**Comment by sinsukehlab** [comment, marked as answer]
You haven't done the task for Step 0. You should first generate a new repository from the template by clicking on the **Start course** button instead of forking the template.

**Reply by blindaks** [reply]
Okay, thank you so much!!

**Reply by sinsukehlab** [reply]
I'm glad that I could be of help.
